test/functional/Java8andUp/src/j9vm/test/loadLibrary/TestLoadLegacyLibrary.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package j9vm.test.loadLibrary;
+
+import org.testng.annotations.Test;
+import org.testng.AssertJUnit;
+
+@Test(groups = { "level.sanity" })
+public class TestLoadLegacyLibrary {
+
+	private static native boolean fooImpl();
+
+	@Test
+	public static void test_load_legacy_library() throws Exception {
+		System.loadLibrary("loadLibraryTest"); //$NON-NLS-1$
+
+		AssertJUnit.assertTrue("Native Method fooImpl failed to return expected result", fooImpl()); //$NON-NLS-1$
```

Comment:
We can use testng assert `org.testng.Assert` `Assert.assertTrue()` 
https://static.javadoc.io/org.testng/testng/6.8.17/org/testng/Assert.html

Reply:
updated